DEPENDENCIES.md

```diff
@@ -34,6 +34,7 @@ following Free and Open Source software:
     github.com/datawire/envconfig                                                v0.0.0-20221012222025-09524dc7d59b        Apache License 2.0
     github.com/datawire/go-ftpserver                                             v0.1.1                                    Apache License 2.0
     github.com/datawire/go-fuseftp/rpc                                           v0.2.0                                    Apache License 2.0
+    github.com/datawire/k8sapi                                                   v0.1.1-0.20221102212418-ff24519a45be      Apache License 2.0
```

Comment:
I'd like to avoid introducing this package until we also remove the existing one.